test/bdd/bddtests_test.go

```diff
@@ -95,10 +95,14 @@ func FeatureContext(s *godog.Suite) {
 		panic(fmt.Sprintf("Error returned from NewBDDContext: %s", err))
 	}
 
+	// set dynamic args
+	context.DynamicArgs[SideTreeURL] = "http://localhost:48326/.sidetree/document"
```

Comment:
I don't think that `Dynamic` is adding to the name :)

Reply:
done